js/components/generate/GenerateForm.js

```diff
@@ -1,16 +1,21 @@
 import React from "react"
-import { Field, Form, Formik } from "formik"
+import { ErrorMessage, Field, Form, Formik } from "formik"
 import { Graph, populateHybridRelatives } from "../graph/Graph"
 import Disclaimer from "../common/Disclaimer"
-import { ErrorMessage } from "../common/react_modal.js.jsx"
+
+const errorMessageStyle = {
```

Comment:
Add this to the CSS file (targeting an error message class; if Formik doesn't provide one by default, you can add your own)